pkg/controller/certificates/trigger/trigger_controller.go

```diff
@@ -371,9 +373,4 @@ func (c *controllerWrapper) Register(ctx *controllerpkg.Context) (workqueue.Type
 }
 
 func init() {
-	controllerpkg.Register(ControllerName, func(ctx *controllerpkg.ContextFactory) (controllerpkg.Interface, error) {
-		return controllerpkg.NewBuilder(ctx, ControllerName).
-			For(&controllerWrapper{}).
-			Complete()
-	})
-}
+	controllerpkg.Register(ControllerName, func(ctx *controllerpkg.ContextF
```

Comment:
There seems to be something missing here, or?

Reply:
Hmm. should be fixed now; might have been a miss click